src/youbora.js

```diff
@@ -53,6 +53,9 @@ class Youbora extends BasePlugin {
     if (this._youbora.getAdapter()) {
       this._youbora.getAdapter().config = update;
     }
+    if (update.options.customAdsAdapter) {
+      this._youbora.setAdsAdapter(config.customAdsAdapter);
```

Comment:
@jesuslc91 this will cause the original ads adapter to also be registered, is this ok?

Reply:
I also think this is a bit confusing - cause if you supply the customAdsAdapter in the config it will override the default one, but if you update the config on the fly it will register it along side the default one.
We should supply a consistent option scheme.
An option can be we have customAdsAdapter option that will always be registered along side the default one, and then you can add another option to disable default ads adapter, but then you also need to supply a way to support unsetting ads adapter if you update it on the fly.

Reply:
We control this on the library, when we have an adapter/ads adapter registered but we register a new one, we remove the previous one: https://bitbucket.org/npaw/lib-plugin-js/src/71ad3b1ed919c89fb70d72b671e1aa052c264bf0/src/plugin/plugin%2Bads.js#lines-26

Why we should do this this way: I was thinking about the case when a customer creates the player first and then creates the ad object. But we don't know if they are going to use the player for ads, or the external object: so by default the constructor sets the default ads adapter if there is no custom one coming, and then you can replace it with the option.

To me makes sense. What do you think?

Reply:
Sounds good to me - raised this as this wasn't apparent from the code here